geoportal/c2cgeoportal_geoportal/scaffolds/create/{{cookiecutter.project}}/build

```diff
@@ -5,21 +5,31 @@ import os
 import os.path
 import platform
 import re
+import shlex
 import shutil
 import stat
 import subprocess
 import sys
 import urllib.request
-from typing import Any, List
+from typing import TYPE_CHECKING, Any, List, Optional
 
 import yaml
 
+CompletedProcess = subprocess.CompletedProcess[str] if TYPE_CHECKING else subprocess.CompletedProcess
 
-def run(args: argparse.Namespace, command: List[str], **kwargs: Any) -> None:
+
+def run(
+    args: argparse.Namespace, command: List[str], exit_on_error: bool = False, **kwargs: Any
+) -> Optional[CompletedProcess]:
     if args.verbose or args.dry_run:
-        print(" ".join(command))
```

Comment:
You do not print the command anymore in case of dry-run ?

Reply:
Oups, missing print...